### PR TITLE
New package: Lenovo.RealtekAudioDriver version 6.0.9159.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111967&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
image

File: manifests/l/Lenovo/RealtekAudioDriver/6.0.9159.1/Lenovo.RealtekAudioDriver.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Lenovo.RealtekAudioDriver
+PackageVersion: 6.0.9159.1
+PackageLocale: en-US
+Publisher: Lenovo Group Limited
```
Comment (Trenly) [inline manifests/l/Lenovo/RealtekAudioDriver/6.0.9159.1/Lenovo.RealtekAudioDriver.locale.en-US.yaml:7]:
```suggestion
Publisher: Realtek Semiconductor Corp.
```

File: manifests/l/Lenovo/RealtekAudioDriver/6.0.9159.1/Lenovo.RealtekAudioDriver.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Lenovo.RealtekAudioDriver
+PackageVersion: 6.0.9159.1
+PackageLocale: en-US
+Publisher: Lenovo Group Limited
+PackageName: Realtek Audio Driver
+License: Proprietary
+Copyright: Copyright © Lenovo 2005 - 2022.
```
Comment (Trenly) [inline manifests/l/Lenovo/RealtekAudioDriver/6.0.9159.1/Lenovo.RealtekAudioDriver.locale.en-US.yaml:10]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**